src/components/media-player/media-player.component.spec.ts

```diff
@@ -4,23 +4,20 @@ import { MediaPlayerModule } from './media-player.module';
 
 @Component({
     selector: 'ux-media-player-audio',
-    template: `<ux-media-player *ngIf="type === 'audio'" class="media-player" type="audio" [displayName]="displayName" [source]="audioSource"></ux-media-player>
+    template: `<ux-media-player class="media-player" [type]="'audio'" [filename]="filename" [source]="audioSource"></ux-media-player>
```

Comment:
Whenever the value for an input is a constant string, you can omit the the `[` `]`, so in this instance you can write:

```
type="audio"
```

If you have square brackets then Angular has to check if the property has changed every time change detection is run, if we know the value will never change then providing a string literal mean Angular has one less thing to check.